BSB_LAN/include/get_cmdtbl_values.h

```diff
@@ -1,65 +1,25 @@
 
-uint32_t get_cmdtbl_cmd(int i) {
-  uint32_t c = 0;
-  int entries1 = sizeof(cmdtbl1)/sizeof(cmdtbl1[0]);
-  int entries2 = sizeof(cmdtbl1)/sizeof(cmdtbl1[0]) +  sizeof(cmdtbl2)/sizeof(cmdtbl2[0]);
-  if (i < entries1) {
+inline uint32_t get_cmdtbl_cmd(int i) {
```

Comment:
Just out of curiosity: Why is `inline` necessary here?

Reply:
In the old books, there was a recommendation to do small functions "inline" since the overhead of calling a function was more than just embedding a piece of code. I tried both options today - there is no difference. It seems that the compiler itself decides when to embed the code and when to make a function call.

Reply:
Thanks! Again learned something new (or old ;) )